features/package-install.feature

```diff
@@ -1,5 +1,35 @@
 Feature: Install WP-CLI packages
 
+  Scenario: Install a package with an old package index url in package composer.json
+    Given an empty directory
+    And a composer.json file:
+      """
+      {
+        "repositories": {
+          "wp-cli": {
+            "type": "composer",
+            "url": "http://wp-cli.org/package-index/"
+          }
+        }
+      }
+      """
+    When I run `rm -rf /tmp/wp-cli-package-install-test/ && mkdir /tmp/wp-cli-package-install-test/ && mv composer.json /tmp/wp-cli-package-install-test/`
+    Then the /tmp/wp-cli-package-install-test/composer.json file should exist
+    When I run `WP_CLI_PACKAGES_DIR=/tmp/wp-cli-package-install-test/ wp --info`
```

Comment:
Instead of depending on `/tmp/`, can't we just put `WP_CLI_PACKAGES_DIR` in a directory relative to the test directory? This way, we don't have to clean up the temp directory at the end of the test run either.